simulators/mujoco_simulator/revolve2/simulators/mujoco_simulator/_scene_to_model.py

```diff
@@ -33,11 +34,12 @@
 
 
 def scene_to_model(
-    scene: Scene, simulation_timestep: float
+    scene: Scene, simulation_timestep: float, cast_shadows: bool = False
```

Comment:
let's remove the default and just set it to false in `simulate_scene`. same behaviour for now but I would like to have little default in the internal code because it can be confusing later